frame/whitelist/src/lib.rs

```diff
@@ -94,9 +94,9 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		CallWhitelisted { call_hash: T::Hash },
-		WhitelistedCallRemoved { call_hash: T::Hash },
-		WhitelistedCallDispatched { call_hash: T::Hash, result: DispatchResultWithPostInfo },
+		CallWhitelisted { call_hash: Hash },
+		WhitelistedCallRemoved { call_hash: Hash },
+		WhitelistedCallDispatched { call_hash: Hash, result: DispatchResultWithPostInfo },
```

Comment:
Maybe you can change these to `H256`  to easier spot that it is **not** the standard `T::Hash`.

Reply:
updated.